api-client/src/routes/index.ts

```diff
@@ -26,6 +26,9 @@ export const OFFERS_ROUTE: OffersRouteType = 'offers'
 export type SprungbrettOfferRouteType = 'sprungbrett'
 export const SPRUNGBRETT_OFFER_ROUTE: SprungbrettOfferRouteType = 'sprungbrett'
 
+export type MalteHelpFormOfferRouteType = 'hilfebutton'
+export const MALTE_HELP_FORM_OFFER_ROUTE: MalteHelpFormOfferRouteType = 'hilfebutton'
```

Comment:
Shouldn't we use an english term here?

Reply:
I don't really have an opinion on that. Do you think just `/help` works better?

Reply:
I'd say so, yes. Depends a little if this is like an official term used in communication (like e.g. sprungbrett) or just an internal name.